.github/workflows/ci-core.yml

```diff
@@ -779,5 +779,6 @@ jobs:
             --go-mod ./go.mod \
             --plugin-file ./plugins/plugins.public.yaml \
             --module github.com/smartcontractkit/chainlink-data-streams \
+            --module github.com/smartcontractkit/chainlink-evm \
```

Comment:
We only need this for remotely installed loops that are set in the `plugins/plugins.*.yaml` manifest files.